src/PokemonGymOwner.java

```diff
@@ -0,0 +1,38 @@
+import java.util.List;
+
+public class PokemonGymOwner extends PokemonTrainer {
+
+//    8 variables (6 in the list)
+    private String name;
+    private String town;
+    private static List<Pokemon> pokemons;
+
+//    1 constructor
+    public PokemonGymOwner(String name, String town, List<Pokemon> pokemons) {
+        super(name, pokemons);
+        this.town = town;
+        System.out.println(this.getName() + this.town + this.getPokemons());
+    }
+
+//      getters
+    public String getName() {
+        return name;
```

Comment:
Nu verwijst "return name" naar de name in deze klasse, maar die is altijd null omdat je de waarde in de constructor naar de super doorstuurt. Als je de "name" variabele uit deze klasse weg haalt, zal "return name" heir gaan wijzen naar de "name" van de super en zal het niet meer null zijn